cccatalog-api/cccatalog/urls.py

```diff
@@ -47,14 +47,122 @@
 an API key as soon as possible. Authorized clients have a higher rate limit
 of 10000 requests per day and 100 requests per minute. Additionally, Creative
 Commons can give your key an even higher limit that fits your application's
-needs. See the `/v1/auth_tokens/register` endpoint for instructions on obtaining
+needs. See the Register and Authenticate section for instructions on obtaining
 an API key.
 
-Pull requests are welcome!
-[Contribute on GitHub](https://github.com/creativecommons/cccatalog-api)
+# Register and Authenticate
 
-"""  # noqa
+## Register for a key
+Before using the CC Catalog API, you need to register access via OAuth2. 
+This can be done using the `/v1/auth_tokens/register` endpoint. 
+
+<br>
+Example on how to register for a key
+
+```
+$ curl -X POST -H "Content-Type: application/json" -d '{"name": "My amazing project", "description": "To access CC Catalog API", "email": "cccatalog-api@creativecommons.org"}' https://api.creativecommons.engineering/v1/auth_tokens/register
+```
+
+<br>
+If your request is succesful, you will get a `client_id` and `client_secret`.
+
+Example of successful request
+
+```
+{
+    "client_secret" : "YhVjvIBc7TuRJSvO2wIi344ez5SEreXLksV7GjalLiKDpxfbiM8qfUb5sNvcwFOhBUVzGNdzmmHvfyt6yU3aGrN6TAbMW8EOkRMOwhyXkN1iDetmzMMcxLVELf00BR2e",
+    "client_id" : "pm8GMaIXIhkjQ4iDfXLOvVUUcIKGYRnMlZYApbda",
+    "name" : "My amazing project"
+}
+```
+
+## Authenticate
+In order to use the CC Catalog API endpoints, you need to include access token \
+in the header.
+This can be done by exchanging your client credentials for a token using the \
+`v1/auth_tokens/token/` endpoint.
+
+<br>
+Example on how to authenticate using OAuth2
+
+```
+$ curl -X POST -d "client_id=pm8GMaIXIhkjQ4iDfXLOvVUUcIKGYRnMlZYApbda&client_secret=YhVjvIBc7TuRJSvO2wIi344ez5SEreXLksV7GjalLiKDpxfbiM8qfUb5sNvcwFOhBUVzGNdzmmHvfyt6yU3aGrN6TAbMW8EOkRMOwhyXkN1iDetmzMMcxLVELf00BR2e&grant_type=client_credentials" https://api.creativecommons.engineering/v1/auth_tokens/token/
+```
+
+<br>
+If your request is successful, you will get an access token.
+
+Example of successful request
+
+```
+ {
+    "access_token" : "DLBYIcfnKfolaXKcmMC8RIDCavc2hW",
+    "scope" : "read write groups",
+    "expires_in" : 36000,
+    "token_type" : "Bearer"
+ }
+```
+
+Check your email for a verification link. After you have followed the link, \
+your API key will be activated.
+
+## Using Access Token
+Include the `access_token` in the authorization header to use your key in \
+your future API requests.
+
+<br>
+Example
+
+```
+$ curl -H "Authorization: Bearer DLBYIcfnKfolaXKcmMC8RIDCavc2hW" https://api.creativecommons.engineering/v1/images?q=test
+```
+<br>
+<blockquote>
+   <b>NOTE :</b> Your token will be throttled like an anonymous user \
+   until the email address has been verified.
+</blockquote>
+
+# Glossary
```

Comment:
A glossary is a good idea, but I would like to see a few minor changes here. For one, since most of these terms will be known to people who consume APIs, it may be best to put the glossary at the end of the document instead of the very beginning so people don't have to scroll so far to get to actual endpoints.

Another issue is that there is a lot of negative space surrounding the terms. Please experiment with some other header sizes (h2 is pretty big).
![glossary_air](https://user-images.githubusercontent.com/25728015/96924137-35728880-1480-11eb-8656-618922c075ea.png)

Reply:
I've changed the headings for glossary contents from h2 to h4.

![image](https://user-images.githubusercontent.com/10581923/97262548-0d0dd400-185c-11eb-871a-6ade4dc0123e.png)

But I don't know how to move the glossary section to the end of document.